InterfacePython/testCommon.py

```diff
@@ -137,6 +143,26 @@ def compareGrids(self, gridA, gridB, bTestRuleNames = True):
         pB = gridB.getConformalTransformASIN()
         np.testing.assert_equal(pA, pB, "Conformal transform ASIN not equal", True)
 
+    def compareCustomTabulated(self, ctA, ctB):
+        '''
+        Compaes two CustomTabulated instances by checking nodes, weights, and other metadata.
+
+        The test passes if the two instances are mathematically identical.
+        '''
+        # Test metadata.
+        self.assertEqual(ctA.getDescription(), ctB.getDescription(), "error in getDescription()")
+        self.assertEqual(ctA.getNumLevels(), ctB.getNumLevels(), "error in getNumLevels()")
+        if (ctA.getNumLevels() == 0): # Empty CustomTabulated instance. Nothing else to check.
```

Comment:
No need to `return`, if `getNumLevels()` is zero then the for-loop range is zero and nothing will happen.

Reply:
Changed.